swc-uxp-starter/README.md

```diff
@@ -0,0 +1,168 @@
+   "enableSWCSupport": true
+}
+```
+
+### **debug.json**
+Contains additionall (and optional) debug config for UDT. You can specify the port number and set the option to `breakOnStart`.
+
+### **src/index.html**
+Includes index.js.
+
+Observe that the SWC components are wrapped with a ```<sp-theme>``` element. This ensures that the Spectrum design tokens are delivered to the scoped HTML context.
+
+### **src/index.js**
+Import the components and themes to deliver the right styling to the components.
+```javascript
+//sp-theme
+import '@spectrum-web-components/theme/sp-theme.js';
+//themes to use classic and spectrum themes
+import '@spectrum-web-components/theme/src/themes.js';
+import '@spectrum-web-components/theme/src/express/themes.js';
+```
+# **Add a new component**
+You can use any of the supported components listed below by following these steps:
+1. Use yarn add to install the components and its supported version.
+
+```javascript
+yarn add @spectrum-web-components/link@0.14.1;
+```
+
+**Note:** Make sure to install the recommended version and add the same under the resolutions block in package.json too.
+
+<img width="800" alt="package.json with resolutions block" src="assets/resolutions-block.png">
+
+
+2. Import the component in *index.js* file.
+
+```javascript
+import '@spectrum-web-components/link/sp-link.js';
+```
+
+3. Add the component in the html file.
+
+```html
+This is an <sp-link href="#">example link</sp-link>.
+```
+
+4. Run `yarn build` or `yarn watch` to prepare the dist bundle.
+
+# **4. Frequently Asked Questions**
+1. Do I need to install specified versions of the components?
+
+   Yes. It is recommended to use the specified versions. There could be inter dependancy of the components and one component could get impacted if the other upgrades. 
+
+2. Do I need to wrap all the components in <sp-theme> ?
+
+   Yes. `<sp-theme>` is the one which delivers the spectrum style tokens to the wrapped html.
+   Also make sure to import the appropriate theme.
+
+3. Why do I need the resolution block in the package.json?
+
+   In order to resolve transitive dependancies we need to [force resolutions](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/). Thereby to avoid incompatibility issues in sub components, one needs to mention all the dependancies in the resolution block too.
+4. I see the following error in the console. How do I resolve this ?
+
+```
+Failed to execute 'define' on 'CustomElementRegistry': the name "sp-icon-alert" has already been used with this registry
+```
+
+Such errors occur when the package manager has resolved more than one copy of the component (sp-icon-alert in this case. Some of the ways to troubleshoot are:
+
+* Make sure you dont have multiple calls to bundle.js somehow.
+* Check if you have added the component in the resolutions block in package.json.
+* Check if you are using correct recommended version of the package without the `^`.
+
+
+
+
+# **Supported Components**
```

Comment:
We are documenting this online. Let's not repeat.
'Visit our online docs (https://developer.adobe.com/photoshop/uxp/2022/uxp-api/reference-spectrum/swc) to find the list of supported components '